### ISSUE TITLE
Why choose the last epoch’s accuracy as final validation accuracy in the validation stage?

### ISSUE THREAD

**Issue by aubest** [Project-MONAI/research-contributions#40]
HI, When I was trying to train the UNETR, I found that the last epoch result was always selected as validation accuracy, But I am confused why not to take the average?
![image](https://user-images.githubusercontent.com/28101192/156907848-94dfb3f6-370f-448a-a24a-d1b30391ccb3.png)

**Comment by ahatamiz** [comment]
Hi @aubest 

Thanks for the comment. After each epoch, we report the validation accuracy of the epoch and once training is finished, we report the best validation accuracy. We also keep the last epoch checkpoint as well as checkpoint with best accuracy on the validation set.
